tests/tcp.c

```diff
@@ -26,9 +26,11 @@
 #include "../src/pair.h"
 #include "../src/pubsub.h"
 #include "../src/tcp.h"
+#include "../src/utils/err.h"
```

Comment:
This file has only whitespace changes now.  These are not intrinsic to you changes, so lets undo this.